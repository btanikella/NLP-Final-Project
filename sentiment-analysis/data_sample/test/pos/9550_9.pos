WRB NNS VBN CD TO NN JJ VBD IN PRP VBD DT RBS JJ NN RB TO VB PRP$ JJ NN
WRB JJ NN NN TO DT NNS IN PRP$ JJ NN JJ VBD JJ IN DT JJ NN IN PRP$ JJ NNS IN DT JJR NN
WRB JJ NN  CC DT JJ NN IN RB JJ CC JJ NNS : IN DT NN IN JJ NN : NN RB  IN DT NN IN CD JJ NNS IN NNP CD  JJ VBD IN NN RBR JJ VBD RB VBN VBN : RB RBR IN NN  TO DT NNS IN RB  VBP DT JJ NN CC VB RP WRB DT NN VBZ CC DT JJ CC JJ NN
CC IN DT VBZ RB NN  NN IN CC VB CD IN : PRP VBZ RB NN PRP VBD NNS VB IN DT JJ NN IN NN
