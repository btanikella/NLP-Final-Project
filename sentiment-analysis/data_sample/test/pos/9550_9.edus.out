Sentence #1 (22 tokens):
when fellini committed 8 1/2 to film many commented that it was the most personal vision ever to find its perfect expression.
[Text=when CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=WRB] [Text=fellini CharacterOffsetBegin=9 CharacterOffsetEnd=16 PartOfSpeech=NNS] [Text=committed CharacterOffsetBegin=17 CharacterOffsetEnd=26 PartOfSpeech=VBN] [Text=8 1/2 CharacterOffsetBegin=27 CharacterOffsetEnd=32 PartOfSpeech=CD] [Text=to CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=TO] [Text=film CharacterOffsetBegin=36 CharacterOffsetEnd=40 PartOfSpeech=NN] [Text=many CharacterOffsetBegin=41 CharacterOffsetEnd=45 PartOfSpeech=JJ] [Text=commented CharacterOffsetBegin=46 CharacterOffsetEnd=55 PartOfSpeech=VBD] [Text=that CharacterOffsetBegin=56 CharacterOffsetEnd=60 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=61 CharacterOffsetEnd=63 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=64 CharacterOffsetEnd=67 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=68 CharacterOffsetEnd=71 PartOfSpeech=DT] [Text=most CharacterOffsetBegin=72 CharacterOffsetEnd=76 PartOfSpeech=RBS] [Text=personal CharacterOffsetBegin=77 CharacterOffsetEnd=85 PartOfSpeech=JJ] [Text=vision CharacterOffsetBegin=86 CharacterOffsetEnd=92 PartOfSpeech=NN] [Text=ever CharacterOffsetBegin=93 CharacterOffsetEnd=97 PartOfSpeech=RB] [Text=to CharacterOffsetBegin=98 CharacterOffsetEnd=100 PartOfSpeech=TO] [Text=find CharacterOffsetBegin=101 CharacterOffsetEnd=105 PartOfSpeech=VB] [Text=its CharacterOffsetBegin=106 CharacterOffsetEnd=109 PartOfSpeech=PRP$] [Text=perfect CharacterOffsetBegin=110 CharacterOffsetEnd=117 PartOfSpeech=JJ] [Text=expression CharacterOffsetBegin=118 CharacterOffsetEnd=128 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=128 CharacterOffsetEnd=129 PartOfSpeech=.] 
root(ROOT-0, committed-3)
advmod(committed-3, when-1)
nsubj(committed-3, fellini-2)
dobj(committed-3, 8 1/2-4)
aux(film-6, to-5)
infmod(8 1/2-4, film-6)
nsubj(commented-8, many-7)
rcmod(film-6, commented-8)
mark(vision-15, that-9)
nsubj(vision-15, it-10)
cop(vision-15, was-11)
det(vision-15, the-12)
amod(vision-15, most-13)
amod(vision-15, personal-14)
ccomp(commented-8, vision-15)
advmod(vision-15, ever-16)
aux(find-18, to-17)
xcomp(vision-15, find-18)
poss(expression-21, its-19)
amod(expression-21, perfect-20)
dobj(find-18, expression-21)
punct(committed-3, .-22)

Sentence #2 (27 tokens):
when dante cast betrice to the flames of his undying passion many were aghast at the honest brutality of his unrequited feelings for a younger woman.
[Text=when CharacterOffsetBegin=134 CharacterOffsetEnd=138 PartOfSpeech=WRB] [Text=dante CharacterOffsetBegin=139 CharacterOffsetEnd=144 PartOfSpeech=JJ] [Text=cast CharacterOffsetBegin=145 CharacterOffsetEnd=149 PartOfSpeech=NN] [Text=betrice CharacterOffsetBegin=150 CharacterOffsetEnd=157 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=158 CharacterOffsetEnd=160 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=161 CharacterOffsetEnd=164 PartOfSpeech=DT] [Text=flames CharacterOffsetBegin=165 CharacterOffsetEnd=171 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=172 CharacterOffsetEnd=174 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=175 CharacterOffsetEnd=178 PartOfSpeech=PRP$] [Text=undying CharacterOffsetBegin=179 CharacterOffsetEnd=186 PartOfSpeech=JJ] [Text=passion CharacterOffsetBegin=187 CharacterOffsetEnd=194 PartOfSpeech=NN] [Text=many CharacterOffsetBegin=195 CharacterOffsetEnd=199 PartOfSpeech=JJ] [Text=were CharacterOffsetBegin=200 CharacterOffsetEnd=204 PartOfSpeech=VBD] [Text=aghast CharacterOffsetBegin=205 CharacterOffsetEnd=211 PartOfSpeech=JJ] [Text=at CharacterOffsetBegin=212 CharacterOffsetEnd=214 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=215 CharacterOffsetEnd=218 PartOfSpeech=DT] [Text=honest CharacterOffsetBegin=219 CharacterOffsetEnd=225 PartOfSpeech=JJ] [Text=brutality CharacterOffsetBegin=226 CharacterOffsetEnd=235 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=236 CharacterOffsetEnd=238 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=239 CharacterOffsetEnd=242 PartOfSpeech=PRP$] [Text=unrequited CharacterOffsetBegin=243 CharacterOffsetEnd=253 PartOfSpeech=JJ] [Text=feelings CharacterOffsetBegin=254 CharacterOffsetEnd=262 PartOfSpeech=NNS] [Text=for CharacterOffsetBegin=263 CharacterOffsetEnd=266 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=267 CharacterOffsetEnd=268 PartOfSpeech=DT] [Text=younger CharacterOffsetBegin=269 CharacterOffsetEnd=276 PartOfSpeech=JJR] [Text=woman CharacterOffsetBegin=277 CharacterOffsetEnd=282 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=282 CharacterOffsetEnd=283 PartOfSpeech=.] 
root(ROOT-0, cast-3)
advmod(cast-3, when-1)
nsubj(cast-3, dante-2)
dobj(cast-3, betrice-4)
prep(betrice-4, to-5)
det(flames-7, the-6)
pobj(to-5, flames-7)
prep(flames-7, of-8)
poss(passion-11, his-9)
amod(passion-11, undying-10)
pobj(of-8, passion-11)
nsubj(aghast-14, many-12)
cop(aghast-14, were-13)
rcmod(passion-11, aghast-14)
prep(aghast-14, at-15)
det(brutality-18, the-16)
amod(brutality-18, honest-17)
pobj(at-15, brutality-18)
prep(brutality-18, of-19)
poss(feelings-22, his-20)
amod(feelings-22, unrequited-21)
pobj(of-19, feelings-22)
prep(feelings-22, for-23)
det(woman-26, a-24)
amod(woman-26, younger-25)
pobj(for-23, woman-26)
punct(cast-3, .-27)

Sentence #3 (76 tokens):
when onur tukel, and a merry band of wickedly talented and misfited filmmakers--from the mecca of movie-making insanity--band together, during the course of two chilly weeks in december 2000, many commented that nothing more sophmoric had ever been committed...much less on film.well, to the doubters out there, watch this original piece and find out why the ID is both a charming and powerful thing.
[Text=when CharacterOffsetBegin=288 CharacterOffsetEnd=292 PartOfSpeech=WRB] [Text=onur CharacterOffsetBegin=293 CharacterOffsetEnd=297 PartOfSpeech=JJ] [Text=tukel CharacterOffsetBegin=298 CharacterOffsetEnd=303 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=303 CharacterOffsetEnd=304 PartOfSpeech=,] [Text=and CharacterOffsetBegin=305 CharacterOffsetEnd=308 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=309 CharacterOffsetEnd=310 PartOfSpeech=DT] [Text=merry CharacterOffsetBegin=311 CharacterOffsetEnd=316 PartOfSpeech=JJ] [Text=band CharacterOffsetBegin=317 CharacterOffsetEnd=321 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=322 CharacterOffsetEnd=324 PartOfSpeech=IN] [Text=wickedly CharacterOffsetBegin=325 CharacterOffsetEnd=333 PartOfSpeech=RB] [Text=talented CharacterOffsetBegin=334 CharacterOffsetEnd=342 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=343 CharacterOffsetEnd=346 PartOfSpeech=CC] [Text=misfited CharacterOffsetBegin=347 CharacterOffsetEnd=355 PartOfSpeech=JJ] [Text=filmmakers CharacterOffsetBegin=356 CharacterOffsetEnd=366 PartOfSpeech=NNS] [Text=-- CharacterOffsetBegin=366 CharacterOffsetEnd=368 PartOfSpeech=:] [Text=from CharacterOffsetBegin=368 CharacterOffsetEnd=372 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=373 CharacterOffsetEnd=376 PartOfSpeech=DT] [Text=mecca CharacterOffsetBegin=377 CharacterOffsetEnd=382 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=383 CharacterOffsetEnd=385 PartOfSpeech=IN] [Text=movie-making CharacterOffsetBegin=386 CharacterOffsetEnd=398 PartOfSpeech=JJ] [Text=insanity CharacterOffsetBegin=399 CharacterOffsetEnd=407 PartOfSpeech=NN] [Text=-- CharacterOffsetBegin=407 CharacterOffsetEnd=409 PartOfSpeech=:] [Text=band CharacterOffsetBegin=409 CharacterOffsetEnd=413 PartOfSpeech=NN] [Text=together CharacterOffsetBegin=414 CharacterOffsetEnd=422 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=422 CharacterOffsetEnd=423 PartOfSpeech=,] [Text=during CharacterOffsetBegin=424 CharacterOffsetEnd=430 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=431 CharacterOffsetEnd=434 PartOfSpeech=DT] [Text=course CharacterOffsetBegin=435 CharacterOffsetEnd=441 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=442 CharacterOffsetEnd=444 PartOfSpeech=IN] [Text=two CharacterOffsetBegin=445 CharacterOffsetEnd=448 PartOfSpeech=CD] [Text=chilly CharacterOffsetBegin=449 CharacterOffsetEnd=455 PartOfSpeech=JJ] [Text=weeks CharacterOffsetBegin=456 CharacterOffsetEnd=461 PartOfSpeech=NNS] [Text=in CharacterOffsetBegin=462 CharacterOffsetEnd=464 PartOfSpeech=IN] [Text=december CharacterOffsetBegin=465 CharacterOffsetEnd=473 PartOfSpeech=NNP] [Text=2000 CharacterOffsetBegin=474 CharacterOffsetEnd=478 PartOfSpeech=CD] [Text=, CharacterOffsetBegin=478 CharacterOffsetEnd=479 PartOfSpeech=,] [Text=many CharacterOffsetBegin=480 CharacterOffsetEnd=484 PartOfSpeech=JJ] [Text=commented CharacterOffsetBegin=485 CharacterOffsetEnd=494 PartOfSpeech=VBD] [Text=that CharacterOffsetBegin=495 CharacterOffsetEnd=499 PartOfSpeech=IN] [Text=nothing CharacterOffsetBegin=500 CharacterOffsetEnd=507 PartOfSpeech=NN] [Text=more CharacterOffsetBegin=508 CharacterOffsetEnd=512 PartOfSpeech=RBR] [Text=sophmoric CharacterOffsetBegin=513 CharacterOffsetEnd=522 PartOfSpeech=JJ] [Text=had CharacterOffsetBegin=523 CharacterOffsetEnd=526 PartOfSpeech=VBD] [Text=ever CharacterOffsetBegin=527 CharacterOffsetEnd=531 PartOfSpeech=RB] [Text=been CharacterOffsetBegin=532 CharacterOffsetEnd=536 PartOfSpeech=VBN] [Text=committed CharacterOffsetBegin=537 CharacterOffsetEnd=546 PartOfSpeech=VBN] [Text=... CharacterOffsetBegin=546 CharacterOffsetEnd=549 PartOfSpeech=:] [Text=much CharacterOffsetBegin=549 CharacterOffsetEnd=553 PartOfSpeech=RB] [Text=less CharacterOffsetBegin=554 CharacterOffsetEnd=558 PartOfSpeech=RBR] [Text=on CharacterOffsetBegin=559 CharacterOffsetEnd=561 PartOfSpeech=IN] [Text=film.well CharacterOffsetBegin=562 CharacterOffsetEnd=571 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=571 CharacterOffsetEnd=572 PartOfSpeech=,] [Text=to CharacterOffsetBegin=573 CharacterOffsetEnd=575 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=576 CharacterOffsetEnd=579 PartOfSpeech=DT] [Text=doubters CharacterOffsetBegin=580 CharacterOffsetEnd=588 PartOfSpeech=NNS] [Text=out CharacterOffsetBegin=589 CharacterOffsetEnd=592 PartOfSpeech=IN] [Text=there CharacterOffsetBegin=593 CharacterOffsetEnd=598 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=598 CharacterOffsetEnd=599 PartOfSpeech=,] [Text=watch CharacterOffsetBegin=600 CharacterOffsetEnd=605 PartOfSpeech=VBP] [Text=this CharacterOffsetBegin=606 CharacterOffsetEnd=610 PartOfSpeech=DT] [Text=original CharacterOffsetBegin=611 CharacterOffsetEnd=619 PartOfSpeech=JJ] [Text=piece CharacterOffsetBegin=620 CharacterOffsetEnd=625 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=626 CharacterOffsetEnd=629 PartOfSpeech=CC] [Text=find CharacterOffsetBegin=630 CharacterOffsetEnd=634 PartOfSpeech=VB] [Text=out CharacterOffsetBegin=635 CharacterOffsetEnd=638 PartOfSpeech=RP] [Text=why CharacterOffsetBegin=639 CharacterOffsetEnd=642 PartOfSpeech=WRB] [Text=the CharacterOffsetBegin=643 CharacterOffsetEnd=646 PartOfSpeech=DT] [Text=ID CharacterOffsetBegin=647 CharacterOffsetEnd=649 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=650 CharacterOffsetEnd=652 PartOfSpeech=VBZ] [Text=both CharacterOffsetBegin=653 CharacterOffsetEnd=657 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=658 CharacterOffsetEnd=659 PartOfSpeech=DT] [Text=charming CharacterOffsetBegin=660 CharacterOffsetEnd=668 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=669 CharacterOffsetEnd=672 PartOfSpeech=CC] [Text=powerful CharacterOffsetBegin=673 CharacterOffsetEnd=681 PartOfSpeech=JJ] [Text=thing CharacterOffsetBegin=682 CharacterOffsetEnd=687 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=687 CharacterOffsetEnd=688 PartOfSpeech=.] 
root(ROOT-0, when-1)
amod(tukel-3, onur-2)
dep(when-1, tukel-3)
punct(tukel-3, ,-4)
cc(tukel-3, and-5)
det(band-8, a-6)
amod(band-8, merry-7)
conj(tukel-3, band-8)
prep(band-8, of-9)
advmod(talented-11, wickedly-10)
amod(filmmakers-14, talented-11)
cc(talented-11, and-12)
conj(talented-11, misfited-13)
pobj(of-9, filmmakers-14)
punct(band-8, ---15)
prep(band-8, from-16)
det(mecca-18, the-17)
pobj(from-16, mecca-18)
prep(mecca-18, of-19)
amod(insanity-21, movie-making-20)
pobj(of-19, insanity-21)
punct(band-8, ---22)
dep(band-8, band-23)
advmod(band-23, together-24)
punct(band-8, ,-25)
prep(band-8, during-26)
det(course-28, the-27)
pobj(during-26, course-28)
prep(course-28, of-29)
num(weeks-32, two-30)
amod(weeks-32, chilly-31)
pobj(of-29, weeks-32)
prep(course-28, in-33)
pobj(in-33, december-34)
num(december-34, 2000-35)
punct(band-8, ,-36)
nsubj(commented-38, many-37)
rcmod(band-8, commented-38)
prep(commented-38, that-39)
dep(that-39, nothing-40)
advmod(sophmoric-42, more-41)
nsubj(committed-46, sophmoric-42)
aux(committed-46, had-43)
advmod(committed-46, ever-44)
auxpass(committed-46, been-45)
rcmod(nothing-40, committed-46)
punct(nothing-40, ...-47)
advmod(less-49, much-48)
advmod(watch-59, less-49)
prep(less-49, on-50)
pobj(on-50, film.well-51)
punct(less-49, ,-52)
prep(less-49, to-53)
det(doubters-55, the-54)
pobj(to-53, doubters-55)
advmod(doubters-55, out-56)
advmod(out-56, there-57)
punct(less-49, ,-58)
dep(nothing-40, watch-59)
det(piece-62, this-60)
amod(piece-62, original-61)
dobj(watch-59, piece-62)
cc(nothing-40, and-63)
conj(nothing-40, find-64)
prt(find-64, out-65)
advmod(is-69, why-66)
det(ID-68, the-67)
nsubj(is-69, ID-68)
advcl(find-64, is-69)
preconj(thing-75, both-70)
det(thing-75, a-71)
amod(thing-75, charming-72)
cc(charming-72, and-73)
conj(charming-72, powerful-74)
nsubj(is-69, thing-75)
punct(when-1, .-76)

Sentence #4 (28 tokens):
and if that doesn't getchya, roll over and rub one out...it's probably time you did so.kudos onur on a great piece of art
[Text=and CharacterOffsetBegin=693 CharacterOffsetEnd=696 PartOfSpeech=CC] [Text=if CharacterOffsetBegin=697 CharacterOffsetEnd=699 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=700 CharacterOffsetEnd=704 PartOfSpeech=DT] [Text=does CharacterOffsetBegin=705 CharacterOffsetEnd=709 PartOfSpeech=VBZ] [Text=n't CharacterOffsetBegin=709 CharacterOffsetEnd=712 PartOfSpeech=RB] [Text=getchya CharacterOffsetBegin=713 CharacterOffsetEnd=720 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=720 CharacterOffsetEnd=721 PartOfSpeech=,] [Text=roll CharacterOffsetBegin=722 CharacterOffsetEnd=726 PartOfSpeech=NN] [Text=over CharacterOffsetBegin=727 CharacterOffsetEnd=731 PartOfSpeech=IN] [Text=and CharacterOffsetBegin=732 CharacterOffsetEnd=735 PartOfSpeech=CC] [Text=rub CharacterOffsetBegin=736 CharacterOffsetEnd=739 PartOfSpeech=VB] [Text=one CharacterOffsetBegin=740 CharacterOffsetEnd=743 PartOfSpeech=CD] [Text=out CharacterOffsetBegin=744 CharacterOffsetEnd=747 PartOfSpeech=IN] [Text=... CharacterOffsetBegin=747 CharacterOffsetEnd=750 PartOfSpeech=:] [Text=it CharacterOffsetBegin=750 CharacterOffsetEnd=752 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=752 CharacterOffsetEnd=754 PartOfSpeech=VBZ] [Text=probably CharacterOffsetBegin=755 CharacterOffsetEnd=763 PartOfSpeech=RB] [Text=time CharacterOffsetBegin=764 CharacterOffsetEnd=768 PartOfSpeech=NN] [Text=you CharacterOffsetBegin=769 CharacterOffsetEnd=772 PartOfSpeech=PRP] [Text=did CharacterOffsetBegin=773 CharacterOffsetEnd=776 PartOfSpeech=VBD] [Text=so.kudos CharacterOffsetBegin=777 CharacterOffsetEnd=785 PartOfSpeech=NNS] [Text=onur CharacterOffsetBegin=786 CharacterOffsetEnd=790 PartOfSpeech=VB] [Text=on CharacterOffsetBegin=791 CharacterOffsetEnd=793 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=794 CharacterOffsetEnd=795 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=796 CharacterOffsetEnd=801 PartOfSpeech=JJ] [Text=piece CharacterOffsetBegin=802 CharacterOffsetEnd=807 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=808 CharacterOffsetEnd=810 PartOfSpeech=IN] [Text=art CharacterOffsetBegin=811 CharacterOffsetEnd=814 PartOfSpeech=NN] 
root(ROOT-0, getchya-6)
cc(getchya-6, and-1)
mark(getchya-6, if-2)
nsubj(getchya-6, that-3)
aux(getchya-6, does-4)
neg(getchya-6, n't-5)
punct(getchya-6, ,-7)
conj(getchya-6, roll-8)
advmod(roll-8, over-9)
cc(getchya-6, and-10)
conj(getchya-6, rub-11)
dobj(rub-11, one-12)
prt(rub-11, out-13)
punct(getchya-6, ...-14)
nsubj(time-18, it-15)
cop(time-18, 's-16)
advmod(time-18, probably-17)
parataxis(getchya-6, time-18)
nsubj(did-20, you-19)
rcmod(time-18, did-20)
nsubj(onur-22, so.kudos-21)
ccomp(did-20, onur-22)
prep(onur-22, on-23)
det(piece-26, a-24)
amod(piece-26, great-25)
pobj(on-23, piece-26)
prep(piece-26, of-27)
pobj(of-27, art-28)

